src/main/java/org/tron/common/runtime/vm/program/InternalTransaction.java

```diff
@@ -92,14 +92,22 @@ public InternalTransaction(Transaction trx,  InternalTransaction.TrxType trxType
       this.transferToAddress = Wallet.generateContractAddress(trx);
       this.note = "create";
       this.value = contract.getNewContract().getCallValue();
+      this.data = contract.getNewContract().getBytecode().toByteArray();
+      if(this.data.length == 0){
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Move this left curly brace to the beginning of next line of code. ([squid:LeftCurlyBraceStartLineCheck](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1633&issues=AWak_C2fdPPRVdwyhXKW&open=AWak_C2fdPPRVdwyhXKW)

[ISSUE_KEY]: AWak_C2fdPPRVdwyhXKW